app/endpoints/calendar.py

```diff
@@ -191,7 +202,24 @@ async def delete_bookings_id(
     db: AsyncSession = Depends(get_db),
     user: models_core.CoreUser = Depends(is_user_a_member_of(GroupType.BDE)),
```

Comment:
Should be changed to as the other endpoints.
```suggestion
    user: models_core.CoreUser = Depends(is_user_a_member),
```